dist/complaint-tool-schema.json

```diff
@@ -39,97 +39,141 @@
-          "minLength": 1,
-          "maxLength": 25
-        },
-        "state": {
-          "type": "string",
-          "enum": [
-            "Alabama",
-            "Alaska",
-            "Arizona",
-            "Arkansas",
-            "California",
-            "Colorado",
-            "Connecticut",
-            "Delaware",
-            "District of Columbia",
-            "Florida",
-            "Georgia",
-            "Hawaii",
-            "Idaho",
-            "Illinois",
-            "Indiana",
-            "Iowa",
-            "Kansas",
-            "Kentucky",
-            "Louisiana",
-            "Maine",
-            "Maryland",
-            "Massachusetts",
-            "Michigan",
-            "Minnesota",
-            "Mississippi",
-            "Missouri",
-            "Montana",
-            "Nebraska",
-            "Nevada",
-            "New Hampshire",
-            "New Jersey",
-            "New Mexico",
-            "New York",
-            "North Carolina",
-            "North Dakota",
-            "Ohio",
-            "Oklahoma",
-            "Oregon",
-            "Pennsylvania",
-            "Rhode Island",
-            "South Carolina",
-            "South Dakota",
-            "Tennessee",
-            "Texas",
-            "Utah",
-            "Vermont",
-            "Virginia",
-            "Washington",
-            "West Virginia",
-            "Wisconsin",
-            "Wyoming"
-          ]
-        },
-        "postalCode": {
-          "type": "string",
-          "pattern": "^\\d{5}$"
-        },
-        "country": {
-          "type": "string",
-          "enum": [
-            "US"
-          ]
-        }
-      }
+      "enumNames": [
+        "Alabama",
+        "Alaska",
+        "Arizona",
+        "Arkansas",
+        "California",
+        "Colorado",
+        "Connecticut",
+        "Delaware",
+        "District Of Columbia",
+        "Florida",
+        "Georgia",
+        "Hawaii",
+        "Idaho",
+        "Illinois",
+        "Indiana",
+        "Iowa",
+        "Kansas",
+        "Kentucky",
+        "Louisiana",
+        "Maine",
+        "Maryland",
+        "Massachusetts",
+        "Michigan",
+        "Minnesota",
+        "Mississippi",
+        "Missouri",
+        "Montana",
+        "Nebraska",
+        "Nevada",
+        "New Hampshire",
+        "New Jersey",
+        "New Mexico",
+        "New York",
+        "North Carolina",
+        "North Dakota",
+        "Ohio",
+        "Oklahoma",
+        "Oregon",
+        "Pennsylvania",
+        "Rhode Island",
+        "South Carolina",
+        "South Dakota",
+        "Tennessee",
+        "Texas",
+        "Utah",
+        "Vermont",
+        "Virginia",
+        "Washington",
+        "West Virginia",
+        "Wisconsin",
+        "Wyoming"
+      ]
+    },
+    "postalCode": {
+      "type": "string",
+      "pattern": "^\\d{5}$"
+    },
+    "country": {
+      "type": "string",
+      "enum": [
+        "US"
```

Comment:
Some [schools](url
https://www.vets.gov/gi-bill-comparison-tool/profile/31100792) are not in the US

Reply:
Thanks @rianfowler, stakeholders indicated that supporting international addresses should be revisited in the future. We can add a comment about that here, and double check with them if of interest.
Another thought relating to this is whether we should display the country field at all if there is only one choice allowed, and simply set this on the BE. But perhaps that is a question for design, and I will ask on the contact page PR.